sdcc/zvb_gfx.c

```diff
@@ -176,6 +177,48 @@ static gfx_error gfx_tileset_load_nibble(gfx_context* ctx, uint8_t* tileset, uin
     return GFX_SUCCESS;
 }
 
+static gfx_error gfx_tileset_load_rle(gfx_context* ctx, uint8_t* data, uint16_t size, uint16_t from, uint8_t pal_offset, uint8_t opacity) {
+    uint8_t buffer[TILE_SIZE + 16] = { 0x00 };
```

Comment:
For some reason, SDCC generates a LOT of instructions because of this declaration (+1KB!)
Consider dropping the `= { 0 }` (the buffer will be set below anyway)

Reply:
removed initializer, reduced lib filesize by almost 7KB!!! That's crazy!